core/scripts/common/helpers.go

```diff
@@ -276,11 +280,26 @@ func FundNodes(e Environment, transmitters []string, fundingAmount *big.Int) {
 	PanicErr(err)
 
 	for i := 0; i < len(transmitters); i++ {
+		// Special case for Arbitrum since gas estimation there is different.
+		var gasLimit uint64
+		if e.ChainID == ArbitrumGoerliChainID {
```

Comment:
Should we just use `IsArbitrumChainID` here instead? It'll check for testnet and mainnet.

Reply:
makes sense. updated

Reply:
`ChainType` is even more general, so that's preferred if it's available: https://github.com/smartcontractkit/chainlink/blob/33ce8d1f3b33557d241067f566edba6fd64bc530/core/config/chaintype.go#L13
Leaves things flexible to test new devnets that pop up in the future.

Reply:
Interesting. We only have `chainID` in this context. Do we have a generic helper method that converts integers to `ChainType`?

Reply:
It would have to come from the chain scoped configuration. Doing a lookup from the chain ID alone is not sufficient.  You can construct a chain scoped config with a general config + chain id. I'm not sure I fully grok the surrounding context of this script though. Maybe not a big deal here